geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableRegistrationDUnitTest.java

```diff
@@ -263,13 +263,7 @@ public void testSimpleDurableClientWithRegistration() {
             0),
         regionName, getClientDistributedSystemProperties(durableClientId, durableClientTimeout)));
 
-    // Send clientReady message
-    durableClientVM.invoke(new CacheSerializableRunnable("Send clientReady") {
-      @Override
-      public void run2() throws CacheException {
-        CacheServerTestUtil.getCache().readyForEvents();
-      }
-    });
```

Comment:
moved this to below the registration. We should registerInterest then indicate ready for events